src/Components/AffectedClustersTable/AffectedClustersTable.spec.ct.js

```diff
@@ -375,60 +386,75 @@ describe('non-empty successful affected clusters table', () => {
   });
 
   describe('sorting', () => {
-    _.zip(['name', 'last_checked_at'], TABLE_HEADERS).forEach(
-      ([category, label]) => {
-        SORTING_ORDERS.forEach((order) => {
-          it(`${order} by ${label}`, () => {
-            const col = `td[data-label="${label}"]`;
-            const header = `th[data-label="${label}"]`;
-
-            cy.get(col).should(
-              'have.length',
-              Math.min(DEFAULT_ROW_COUNT, data.length)
-            );
-            if (order === 'ascending') {
-              cy.get(header).find('button').click();
-            } else {
-              cy.get(header).find('button').dblclick();
-            }
+    _.zip(
+      ['name', 'cluster_version', 'last_checked_at'],
```

Comment:
Maybe we can use here `meta.cluster_version` as in https://github.com/RedHatInsights/ocp-advisor-frontend/blob/master/src/Components/ClustersListTable/ClustersListTable.spec.ct.js#L228 to avoid having to modify the code of the test itself, except for
```if (it.meta.cluster_version === '') {
              it.meta.cluster_version = '0.0.0';
            }```